tests/foreman/ui/test_oscapcontent.py

```diff
@@ -147,6 +147,35 @@ def test_positive_update(self):
             self.assertIsNotNone(
                 self.oscapcontent.search(content_name))
 
+    @skip_if_bug_open('bugzilla', 1490348)
+    @tier2
+    def test_positive_update_org_default_content(self):
+        """Update's Org of default Openscap content
+
+        @id: ef888789-3642-46c5-b35f-4aafc0b91864
+
+        @Steps:
+
+        1. Update the openscap content, here the Org.
+
+        @expectedresults: The org for default Openscap content is updated.
+
+        @BZ: 1490348
+
+        @CaseLevel: Integration
+        """
+        org = entities.Organization(name=gen_string('alpha')).create()
+        with Session(self.browser) as session:
+            set_context(session, org=ANY_CONTEXT['org'])
+            self.oscapcontent.update(
+                OSCAP_DEFAULT_CONTENT['rhel6_content'],
+                content_org=org.name)
+            session.nav.go_to_select_org(org.name)
+            self.assertIsNotNone(
+                self.oscapcontent.search(
+                    OSCAP_DEFAULT_CONTENT['rhel7_content']
```

Comment:
perhaps you mean 
```python
OSCAP_DEFAULT_CONTENT['rhel6_content']
```

Reply:
@san7ket I do not also see assertion that org has been updated for that entity

Reply:
will do this

Reply:
@ldjebran the org is switch and then we search for the content in that org